docs/DafnyRef/Attributes.md

```diff
@@ -419,16 +419,16 @@ There are also two different approaches to executing all tests in a program:
    This runner is currently very basic, but avoids introducing any additional target
    language dependencies in the compiled code.
 
-### 23.2.15. `{:timeLimit N}`
+### 22.2.14. `{:timeLimit N}` {#sec-time-limit}
```

Comment:
This reversion of the section number can't be correct. Are you sure you are in synch with master? or did you add a section?  Run 'make numbers' in DafnyRef to recompute all section numbers.